README.md

```diff
@@ -75,6 +75,8 @@ The preferred method is to use the default AWS credentials pattern.  If no AWS c
       "baseUrlDirect": false, // default value
       "signatureVersion": 'v4', // default value
       "globalCacheControl": null, // default value. Or 'public, max-age=86400' for 24 hrs Cache-Control
+      "presignedUrl": false, // default value
```

Comment:
Maybe we can add a short explanation to the two new parameters in addition to the default value? The other parameters also are mostly without comment, however this is not ideal and should not be considered a model for new parameters. Example syntax:

`<KEY>: <DEFAULT_VALUE>, // <OPTIONAL?> <EXPLANATION> Default is <DEFAULT_VALUE>.`
`"presignedUrl": false, // Optional. Is true if (...). Default is false.`
`"presignedUrlExpires": 300, // Optional. Sets the (...) in seconds. Default is 300 seconds.`

Also, to address the shortcomings of the parameter descriptions, a [parameter table](https://github.com/parse-community/parse-server-s3-adapter/blob/master/README.md#parameters) has been added that will eventually replace these descriptions. Can you please also add the new parameters `presignedUrl`, `presignedUrlExpires` to it?

Reply:
When I went to update README.md for the first time, I saw so many warnings in my markdown lint that I ended up just doing what was necessary. I just made the basic adjustment you recommended.

I prefer to create another PR just to adjust the README.md.

Reply:
Maybe I'm overlooking this on mobile, but can you please add the new parameters to the existing table, regardless of any lint warnings. I would rather see the table being updated with every PR that introduces a new parameter, because the existing descriptions are far from optimal. Any general lint issues can be addressed in a separate PR.

Reply:
As I am not good with English, it may not have been clear in my last comment, **I will create another PR to adjust the README as soon as it is approved**.

There is also another PR that I want to create to solve the https://github.com/parse-community/parse-server-s3-adapter/issues/118 issue, but I am also depending on the approval of that PR.

Reply:
I understood your last comment, but this PR introduces new parameters and therefore I would rather see two rows added to the table with this PR.